packages/frontend-v2/pages/index.tsx

```diff
@@ -10,13 +12,332 @@ const Home: NextPage = () => {
     SeasonEnum.S2,
   ];
 
+  const [token, setToken] = useState("");
+  const [planId, setPlanId] = useState<number>();
+
+  // handler that calls fetchCourse from our api-client package
+  const testFetchCourse = (subject: string, classId: string) => {
+    SearchAPI.fetchCourse(subject.toUpperCase(), classId).then((res) =>
```

Comment:
do we need to uppercase every course when hitting search's api? if so we should move this to the fetchCourse function itslef